apps/helper-app/src/app/register-institution/register-institution.component.scss

```diff
@@ -85,6 +85,6 @@ mat-form-field {
 }
 
 re-captcha {
-  margin-top: 32px;
+  margin: 1rem 0;
```

Comment:
We should use the $basic-spacer in general, nevertheless, just a remark

Reply:
4*$basic-spacer = 32px

Reply:
I wasn't really aware of this variable, but yes it could be used more frequently ;)